Graphs/Table/index.js

```diff
@@ -306,6 +306,8 @@ const TableGraph = (props) => {
                 const columnRow = columns[index];
                 const displayColumn = stateColumn.includes(columnRow.label || columnRow.column) ? 'false' : 'true';
                 const sort = !isEmpty(columnKeys) ? columnKeys.get(columnRow.column) : true;
+                // save typeof column value if additionalSearchOperator is true
+                const columnDataType = !!properties.additionalSearchOperator && !!data.length ? typeof objectPath.get(data[0], columnRow.column) : undefined;
```

Comment:
lets minimize the checks inside this for loop.
pls define a const outside of this loop,
```
const conditionProcessColumnDataType = properties.searchBar && properties.enableNumericSearch && data.length;
```
change this statement within loop as,
```
const columnDataType = conditionProcessColumnDataType ? typeof objectPath.get(data[0], columnRow.column) : undefined;
```